app/enterprise/2.1.x/deployment/installation/amazon-linux-2.md

```diff
@@ -125,7 +125,7 @@ You should now have two files in your home directory on the target Amazon system
 Execute a command similar to the following, using the appropriate RPM file name you downloaded.
```

Comment:
General question: the enterprise install files that should be changed are the ones under `app/enterprise/2.1.x`? For CE, the install files are under `app/install/ubuntu.md` with no version suffix appended to the path file, that's why I am asking.

Reply:
Yep, that's correct.